src/form/src/Form.tsx

```diff
@@ -38,9 +38,10 @@ const formProps = {
   disabled: Boolean,
   size: String as PropType<'small' | 'medium' | 'large'>,
   showRequireMark: {
-    type: [Boolean, String] as PropType<'left' | 'right' | boolean>,
+    type: Boolean,
```

Comment:
```suggestion
    type: Boolean as PropType<boolean | undefined>,
```

@Volankey 这么写 setup 里面的类型才能正确